specification/network/resource-manager/Microsoft.Network/preview/2022-06-01-preview/networkVerifier.json

```diff
@@ -0,0 +1,1530 @@
+            "description": "The name of the Network Verifier resource",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "in": "path",
+            "name": "analysisIntentName",
+            "description": "The name of the Analysis Intent resource",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "List of AnalysisRun",
+            "schema": {
+              "$ref": "#/definitions/AnalysisRunList"
+            }
+          },
+          "default": {
+            "description": "Default error response.",
+            "schema": {
+              "$ref": "#/definitions/ResourceProviderDefaultErrorResponse"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "x-ms-examples": {
+          "AnalysisRunList": {
+            "$ref": "./examples/AnalysisRunList.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Network/networkVerifiers/{networkVerifierName}/analysisIntents/{analysisIntentName}/analysisRuns/{analysisRunName}": {
+      "get": {
+        "tags": [
+          "AnalysisRun"
+        ],
+        "summary": "Gets the specific Analysis Run resource",
+        "operationId": "AnalysisRunGet",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "in": "path",
+            "name": "networkVerifierName",
+            "description": "The name of the Network Verifier resource",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "in": "path",
+            "name": "analysisIntentName",
+            "description": "The name of the Analysis Intent resource",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "in": "path",
+            "name": "analysisRunName",
+            "description": "The name of the Analysis Run resource",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "AnalysisRun",
+            "schema": {
+              "$ref": "#/definitions/AnalysisRun"
+            }
+          },
+          "404": {
+            "description": "AnalysisRun Not Found",
+            "schema": {
+              "$ref": "#/definitions/Error"
+            },
+            "x-ms-error-response": true
+          },
```

Comment:
same here

Reply:
Hi @rkmanda, the developer who opened this PR was an intern who finished his internship last week, can you please review https://github.com/Azure/azure-rest-api-specs/pull/20393 PR which is the same as this one? I can help to address these comments in there.